tests/Integration/goldens/redis/src/V1/Gapic/CloudRedisGapicClient.php

```diff
@@ -347,9 +346,8 @@ public function resumeOperation($operationName, $methodName = null)
      *           default this settings points to the default client config file, which is
      *           provided in the resources folder.
      *     @type string|TransportInterface $transport
-     *           The transport used for executing network requests. May be either the string
-     *           `rest` or `grpc`. Defaults to `grpc` if gRPC support is detected on the system.
-     *           *Advanced usage*: Additionally, it is possible to pass in an already
+     *           The transport used for executing network requests. At the moment, supports only
+     *           `rest`. *Advanced usage*: Additionally, it is possible to pass in an already
```

Comment:
The deprecated clients are still incorrect - We should either make the _same_ changes to `GapicClientGenerator.php`, or _preferrably_ continue to throw the "grpc is not supported" error if V1 clients are requested (e.g. if `transport` is `grpc` AND `migration_mode` is anything other than `NEW_SURFACE_ONLY`, throw an error). If we do the latter, we'll need to update these integration tests to be `NEW_SURFACE_ONLY` so they don't throw an error (or add a new Integration client).

Reply:
@bshaffer 
I am confused, why is the one on the`Client` client folder flipped to the correct one and not this one?
![image](https://github.com/googleapis/gapic-generator-php/assets/9062626/07aee3d9-fac3-47c7-a67e-311f18d7d28f)

Reply:
Because we have both the `GapicClientGenerator` and the `GapicClientV2Generator`. It's admittedly confusing, and will only be there until we release V2 for all our client libraries (which should happen soon).